Makefile

```diff
@@ -303,4 +312,6 @@ clean :
 	-$(RM_DIR) $(PYTHON_BUILD_DIR)
 	-$(RM_DIR) $(DEV_SERVICES_DIR)
 	-$(RM) $(SYMLINK_PLUGINS_DIR)
+# Remove FOGLAMP_VERSION from 'scripts/foglamp'
```

Comment:
Why not simply source the file?

Reply:
We can source the file, but the file VERSION has to be copied via "make install" in a location, which can be $FOGLAMP_ROOT/VERSION.


Not tested with Snap yet